README.md

```diff
@@ -713,6 +713,49 @@ To run the go tests
 make test
 ```
 
+# Verify Proofs
+
+## Gnark Plonk
+
+### Dependencies
+This guide assumes that:
+- Gnark library is installed. If not, install it using the following command inside your Go module:
+```bash
+go get github.com/consensys/gnark@latest
+```
+- gnark project to generate the proofs,  instructions[ here](https://docs.gnark.consensys.io/category/how-to)
```

Comment:
I'm not sure if this guide is the best for generating a proof. Maybe we could add a HackMD 
https://hackmd.io/@xr7wfZ30Q3eKDrtinAJaOQ/B1cp2nV4A